### DISCUSSION TITLE
Documentation for "add_entity" for the KnowledgeBase class

### DISCUSSION THREAD

**Discussion by deepak-shenoy** [explosion/spaCy#10264] [Help: Coding & Implementations] [answered]
I'm trying to use add_entity to create my custom entities using the `KnowledgeBase` class.  The documentation here https://spacy.io/api/kb#add_entity doesn't seem to correspond to the method signature within `kb.py` (internal spaCy code).  I'm trying to add entities and I'm getting the standard Type Error positional arguments error which means that I'm missing something using this method.  I'm trying to get more clarity around the vector type which is also referenced within `kb.pyx` - Can someone point me to the *latest* documentation or examples for `add_entity` as the video examples seem to pass in a `numpy` float array?

**Comment by polm** [comment, marked as answer]
The docs on the site are always the latest docs. Occasionally they get out of sync, but everything looks fine here to me - you can also see that the example there matches [our test cases](https://github.com/explosion/spaCy/blob/5adedb8587818741dcd4ee1364ffb3f7d5074e75/spacy/tests/pipeline/test_entity_linker.py#L62), for example.

If you have code and an error could you share them? That would make it easier to figure out what's wrong.

**Reply by deepak-shenoy** [reply]
I was following along with the code displayed in the following video - https://spacy.io/universe/project/video-entity-linking  After looking at each step closer I see that the libraries have changed.  There seems to be an issue using the numpy library for the vector parameter that's looking for a list of float.
